### DISCUSSION TITLE
Convert JSON to SCSS

### DISCUSSION THREAD

**Discussion by legotail** [six7/figma-tokens#337] [Ideas]
Is there any way that I can convert the Firgma Tokens JSON file to the SCSS file?

**Comment by aeviou** [comment]
Bumping this. This plugin is great but I'm wondering if I have to design my own JSON > SCSS/CSS file or if there's some standard protocol or tool that people are already using?

**Reply by legotail** [reply]
For time being, I am using Style Dictionary to convert JSON to SCSS and other formats too.

**Reply by six7** [reply]
What @uipetals said: Style Dictionary is your best bet! [Here's an example repository](https://github.com/six7/figma-tokens-example) using the GitHub sync of the plugin and a GitHub Action using Style Dictionary to automatically convert the tokens JSON to .css variables. Could just as well be scss!
